src/main/webapp/index.html

```diff
@@ -50,23 +51,20 @@
   <header class="masthead text-center text-white">
     <div class="masthead-content">
       <div class="container">
-        <h1 class="masthead-heading mb-0">Techish</h1>
-        <h2 class="masthead-subheading mb-0">A Student Mentor Matching Application</h2>
-        <a href="#" class="btn btn-primary btn-xl rounded-pill mt-5">Learn More</a>
+        <h1 class="masthead-heading mb-0" style="text-align: left">Techish</h1>
+        <h2 class="masthead-subheading mb-0" style="text-align: left">A Student Mentor Matching Application</h2>
+        <a href="#learn-more" class="btn btn-learnmore btn-xl rounded-pill mt-5">Learn More</a>
       </div>
     </div>
-    <div class="bg-circle-1 bg-circle"></div>
-    <div class="bg-circle-2 bg-circle"></div>
-    <div class="bg-circle-3 bg-circle"></div>
-    <div class="bg-circle-4 bg-circle"></div>
+
   </header>
 
   <section>
     <div class="container">
       <div class="row align-items-center">
         <div class="col-lg-6 order-lg-2">
           <div class="p-5">
-            <img class="img-fluid rounded-circle" src="img/01.jpg" alt="">
+            <img class="img-fluid rounded-circle" src="img/undraw_video_call_kxyp.svg" alt="">
```

Comment:
nit: Could "undraw_video_call_kxyp.svg" be named something more meaningful or descriptive?

Reply:
Done